package.json

```diff
@@ -146,4 +147,4 @@
     "trailingComma": "es5",
     "printWidth": 100
   }
-}
+}
```

Comment:
nit: Needs a EOL new line. (Not sure how this got removed :) )

Reply:
strange I added that new EOL but it is not showing up in the review. 


check [ebfd1ce](https://github.com/aws/amazon-chime-sdk-js/pull/2053/commits/ebfd1ce650e05ce6185dba13609b2ed9e929ba61)

Reply:
I had to use *vim* editor to add the new line. Seems like my other editor was removing the new line.